test/src/099-http_tracing/setup_teardown

```diff
@@ -0,0 +1,21 @@
+test099_mount() {
+  local repo=$1
+  local params=$2
+
+  cvmfs_mount $repo $params
+}
+
+test099_unmount() {
+  local repo=$1
+
+  cvmfs_umount $repo
+}
+
+cleanup() {
+  echo "cleanup $cvmfs_test_name"
+  if [ "x$CVMFS_TEST_099_TMPFILE" != "x" ]; then
+    rm $CVMFS_TEST_099_TMPFILE
```

Comment:
Stuff created in the current working directory does not need cleanup, it's being removed by the test driver.
```suggestion
```